src/lib/y2partitioner/actions/delete_bcache.rb

```diff
@@ -35,6 +35,28 @@ def initialize(*args)
 
     private
 
+      # @see DeleteDevice#errors
+      def errors
+        errors = super + [shared_cset]
+
+        errors.compact
+      end
+
+      # Error when the bcache shares cset with any other bcache.
+      #
+      # @see doc/bcache.md
+      # @return [String, nil] nil if there is no error
+      def shared_cset
+        return nil unless device.bcache_cset
+        return nil if single_bcache_cset? || !device.exists_in_probed?
+
+        # TRANSLATORS: Error when trying to existing bcache that shares caches.
+        _(
+          "The bcache shares cache set with other devices which can result in unreachable space " \
+            "if done without detach. But detach can take in some situation very long time."
```

Comment:
Hard to read. Let me try to suggest an alternative:

This device shares its cache set with another bcache device. It's not safe to delete this device without detaching the cache set first. YaST cannot perform such operation, it may take a very long time.

Reply:
BTW, what if the bcache is actually detached (because the user did it before running the Partitioner). How would that be seen by our probing mechanism? Would the bache device and its detached cset be part of the system devicegraph.

Reply:
@ancorgs detached bcache does not have caching set, so we do not see any caching device. I actually show it during initial review, as one of my testing bcache has detached cache.

Reply:
@ancorgs so in this case, it works as expected as no caching set is there, so nothing blocks removal of bcache itself.

Reply:
and about that cset, probably you will see it in device graph visualization, but nowhere else.

Reply:
Ok. Visualization of the csets is probably another of the topics we have to think about. But not now.

Reply:
you can of course use detached cset for another caching device. We can in fact even can allow to attach it to bcache that missing cache. see how it looks with detached cache set:

![snimek obrazovky_2018-10-16_14-12-45](https://user-images.githubusercontent.com/478871/47015618-c1dd6880-d14d-11e8-9576-792bcee79237.png)